man/oomd.conf.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version='1.0'?> <!--*-nxml-*-->
+<!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+  "http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
+<!-- SPDX-License-Identifier: LGPL-2.1+ -->
+
+<refentry id="oomd.conf" conditional='ENABLE_OOMD'
+    xmlns:xi="http://www.w3.org/2001/XInclude">
+  <refentryinfo>
+    <title>oomd.conf</title>
+    <productname>systemd</productname>
+  </refentryinfo>
+
+  <refmeta>
+    <refentrytitle>oomd.conf</refentrytitle>
+    <manvolnum>5</manvolnum>
+  </refmeta>
+
+  <refnamediv>
+    <refname>oomd.conf</refname>
+    <refname>oomd.conf.d</refname>
+    <refpurpose>Global systemd-oomd configuration files</refpurpose>
+  </refnamediv>
+
+  <refsynopsisdiv>
+    <para><filename>/etc/systemd/oomd.conf</filename></para>
+    <para><filename>/etc/systemd/oomd.conf.d/*.conf</filename></para>
+    <para><filename>/usr/lib/systemd/oomd.conf.d/*.conf</filename></para>
+  </refsynopsisdiv>
+
+  <refsect1>
+    <title>Description</title>
+
+    <para>These configuration files control the various parameters of systemd-oomd, a
+    userspace out of memory (OOM) killer.</para>
+
+  </refsect1>
+
+  <xi:include href="standard-conf.xml" xpointer="main-conf" />
+
+  <refsect1>
+    <title>[Oomd] Section Options</title>
+
+    <para>The following options are available in the <literal>[Oomd]</literal> section:</para>
+
+    <variablelist class='oom-directives'>
+      <varlistentry>
+        <term><varname>ResourceMetrics=</varname></term>
+        <listitem><para>Controls which resource metrics <command>systemd-oomd</command> will use to determine
+        when and which cgroups to kill. Takes a whitespace-separated list of resource names. This option may
```

Comment:
"when to kill processes", or "when to take action"? (This settting is not about which cgroups to kill).